mkdocs.yml

```diff
@@ -137,7 +137,17 @@ nav:
         - microservices/application/GeneralAppServiceConfig.md
         - Application Services:
             - Available Services: microservices/application/services/AvailableAppServices.md
-            - microservices/application/services/AppServiceConfigurable.md
+            - App Service Configurable:
+                - Purpose: microservices/application/services/AppServiceConfigurable/Purpose.md
+                - Getting Started: microservices/application/services/AppServiceConfigurable/GettingStarted.md
+                - Configuration: microservices/application/services/AppServiceConfigurable/Configuration.md
```

Comment:
Did we ever resolve if we can have multiple sections named Purpose or Getting Started? just curious.

Reply:
Yes we can. It it hieratical, so no problem. Only issue would be if we tried to have them at the same level.